vscode/src/statusbar.ts

```diff
@@ -73,40 +73,42 @@ export function activateTargetProfileStatusBarItem(): vscode.Disposable[] {
   return disposables;
 }
 
-function registerTargetProfileCommand() {
+async function setTargetProfile() {
+  const target = await vscode.window.showQuickPick(
+    targetProfiles.map((profile) => ({
+      label: profile.uiText,
+    })),
+    { placeHolder: "Select the QIR target profile" },
+  );
+
+  if (target) {
+    setTarget(getTargetProfileSetting(target.label));
+  }
+}
+
+function registerSetTargetProfileCommand() {
   return vscode.commands.registerCommand(
     `${qsharpExtensionId}.setTargetProfile`,
-    async () => {
-      const target = await vscode.window.showQuickPick(
-        targetProfiles.map((profile) => ({
-          label: profile.uiText,
-        })),
-        { placeHolder: "Select the QIR target profile" },
-      );
-
-      if (target) {
-        setTarget(getTargetProfileSetting(target.label));
-      }
-    },
+    setTargetProfile,
   );
 }
 
 const targetProfiles = [
-  { configName: "base", uiText: "Q#: QIR base" },
-  { configName: "adaptive_ri", uiText: "Q#: QIR Adaptive RI" },
-  { configName: "adaptive_rif", uiText: "Q#: QIR Adaptive RIF" },
-  { configName: "unrestricted", uiText: "Q#: unrestricted" },
+  { configName: "base", uiText: "QIR: base" },
+  { configName: "adaptive_ri", uiText: "QIR: Adaptive RI" },
+  { configName: "adaptive_rif", uiText: "QIR: Adaptive RIF" },
+  { configName: "unrestricted", uiText: "QIR: unrestricted" },
```

Comment:
Same comment as in other places - I thought the names were "QIR base", "QIR Adaptive RI/RIF", and "unrestricted" - the "Q#:" part was just a prefix (which I'm fine with removing)